hugo-extended-needed-for-hugo fresh theme (#80)
StefMa/hugo-fresh: README.md

![Fresh theme logo](images/screenshot.png)

## Requirement 

> hugo extended needed for hugo fresh theme for details see [Hugo Support](https://discourse.gohugo.io/t/hugo-extended-needed-for-hugo-fresh-theme/13844) and can be installed from here in release [hugo](https://github.com/gohugoio/hugo/releases)

Reviewer: ```suggestion
The theme requires the hugo extended version to be installed. 
You can grab this version from [here](https://github.com/gohugoio/hugo/releases).
```
